maven-core/src/main/java/org/apache/maven/project/ProjectBuildingException.java

```diff
@@ -72,7 +67,7 @@ protected ProjectBuildingException( String projectId, String message, File pomFi
 
     public ProjectBuildingException( List<ProjectBuildingResult> results )
     {
-        super( createMessage( results ) );
+        super( "Some problems were encountered while processing the POMs" );
```

Comment:
POMs? Not POM?

Reply:
That's what the original message said (see line 126 in the old code), it's a literal move.

Reply:
I guess the message applies to the entire reactor, doesn't it? If so, leave as-is.

Reply:
> I guess the message applies to the entire reactor, doesn't it?

It does. I'll consider this resolved.